desktop/components/react/responsive_window/index.js

```diff
@@ -45,7 +45,7 @@ class ResponsiveWindow extends Component {
     // avoiding race conditions. Could possibly be fixed in React 16.
     setTimeout(() => {
       this.props.onChange(window.innerWidth)
-    }, 1)
+    }, 50)
```

Comment:
Need to rewrite that clock in React, but I suspect this weird race condition will be fixed once we upgrade to latest version.